src/testRouter.js

```diff
@@ -28,8 +28,8 @@ const isSupportedContentType = (contentType) => {
 
 const testRouter = new Router({ prefix: '/test-router' });
 
-const getDestHandler = (version, destination) =>
-  require(`./${version}/destinations/${destination}/transform`);
+const getDestHandler = (versionNo, destination) =>
```

Comment:
Currently version is not a number but rather a string as `v0`. So I don't think the variable `versionNo` makes sense. Let me know if you think the same.

Reply:
changed to ver, version is giving lint issue as version is declared in outer scope